tests/core/test_core.py

```diff
@@ -553,22 +553,22 @@ def test_maximum_popup_dimensions(
     @pytest.mark.parametrize(
         "active_conversation_info",
         [
-            case({"sender_name": "hamlet"}, id="in_pm_narrow_with_sender_typing:start"),
+            case({"hamlet"}, id="in_pm_narrow_with_sender_typing:start"),
             case({}, id="in_pm_narrow_with_sender_typing:stop"),
         ],
     )
     def test_show_typing_notification(
         self,
         mocker: MockerFixture,
         controller: Controller,
-        active_conversation_info: Dict[str, str],
+        active_conversation_info: Set[Any],
     ) -> None:
         set_footer_text = mocker.patch(VIEW + ".set_footer_text")
         sleep = mocker.patch(MODULE + ".time.sleep")
         controller.active_conversation_info = active_conversation_info
 
         def mock_typing() -> None:
-            controller.active_conversation_info = {}
+            controller.active_conversation_info = set()
```

Comment:
In Line 571 and Line 589 ,The `{}` works for test whereas `set()` works for Lint